configure.ml

```diff
@@ -919,16 +919,19 @@ let datadir,datadirsuffix = let (_,_,d,s) = select "DATADIR" in d,s
 
 let cflags_dflt = "-Wall -Wno-unused -g -O2 -fexcess-precision=standard"
 
-let cflags_sse2 = ["-msse2"; "-mfpmath=sse"]
+let cflags_sse2 = "-msse2 -mfpmath=sse"
 
 let cflags, sse2_math =
   let _, slurp =
     (* Test SSE2_MATH support <https://stackoverflow.com/a/45667927> *)
-    tryrun "cc" (["-march=native"; "-dM"; "-E"]
-                 @ cflags_sse2
-                 @ [coqtop/"kernel/byterun/coq_interp.h"] (* any file *)) in
+    tryrun camlexec.find
+      (["ocamlc"]
+       @ List.fold_right (fun o l -> "-ccopt" :: o :: l)
+           (String.split_on_char ' '
```

Comment:
Note that you can pass all the options at once to `-ccopt`. In other words, `["ocamlc"; "-ccopt"; (cflags_dflt ^ " -march=native -dM -E " ^ cflags_sse2); "-c"; ...]` should be fine. No need to split `CFLAGS` and duplicate `-ccopt`.

Reply:
That's the first thing I tried. Unfortunately, as I said above, it didn't work (the C compiler was receiving `native -dM -E`... as a single argument for `march`, don't know what happened to the spaces, I didn't investigate further as duplicating `-ccopt` sounded like a reasonnable fix).

Reply:
Oh, I hadn't understood what you meant by "to duplicate the `-ccopt` options" (despite both of us using the exact same terminology). That is quite mindboggling that it works from command line and `Makefile` but does not from `configure.ml`.

Reply:
Maybe `Unix.create_process` is replacing spaces by unbreakable spaces or something like that. I don't know precisely.

Reply:
That's strange, when I try eg `Unix.create_process "ocamlfind" [|"ocamlfind";"ocamlc";"-ccopt";"-march=native -dM -E";"-c";"dev/header.c"|] Unix.stdin Unix.stdout Unix.stderr` it works fine

Reply:
Indeed, my mistake was to try to protect the argument with `"` or `'` (i.e. using
```OCaml
Unix.create_process "ocamlfind" [|"ocamlfind";"ocamlc";"-ccopt";"\"-march=native -dM -E\"";"-c";"dev/header.c"|] Unix.stdin Unix.stdout Unix.stderr
```
instead of what you offers). Thanks to both of you and sorry for all the noise!